src/sfizz/FlexEnvelope.cpp

```diff
@@ -172,6 +163,10 @@ void FlexEnvelope::Impl::process(absl::Span<float> out)
     const FlexEGDescription& desc = *desc_;
     size_t numFrames = out.size();
     const float samplePeriod = samplePeriod_;
+
+    if (desc.dynamic)
+        updateCurrentTimeAndLevel();
```

Comment:
In my experience, `egN_dynamic` operates at audio rate.
It never feels right to update at buffer intervals, because this is not predictable.
It's better IMO having 2 versions of process, one for static and one for dynamic.